dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ServiceConfig.java

```diff
@@ -102,11 +99,6 @@
      */
     private static final Map<String, Integer> RANDOM_PORT_MAP = new HashMap<String, Integer>();
 
-    /**
-     * A delayed exposure service timer
-     */
-    private static final ScheduledExecutorService delayExportExecutor = Executors.newSingleThreadScheduledExecutor(new NamedThreadFactory("DubboServiceDelayExporter", true));

```

Comment:
It seems that this delayExportExecutor is a global delay exposed component. I don't think we can delete this field.

Since `ServiceConfig` exists, we can export the service at runtime, so we can't determine if `delayExportExecutor` will still be used. In addition, maintaining an object in memory does not have a big impact.

Reply:
Each ServerConfig instance holds its own delayExportExecutor and  delayExportExecutor only used for delay exporting,I feel it is unnecessary to use it.

Reply:
> Each ServerConfig instance holds its own delayExportExecutor and delayExportExecutor only used for delay exporting,I feel it is unnecessary to use it.

agree with @carryxyh , all ServiceConfig holds the same delayExportExecutor

Reply:
My fault,  I didn't notice that it was static.